### DISCUSSION TITLE
Parts and chapters independent on one another

### DISCUSSION THREAD

**Discussion by aburkov** [jgm/pandoc#10040] [Q&A]
Hi!

I'm using Pandoc with rendering into docx. I would like to organize my chapters into several parts. Parts would have numbers like 1, 2, 3, etc. I would like chapters to have numbers independent from parts. Like this:

Part 1

Chapter 1
Section 1.1

Chapter 2
Section 2.1

Part 2

Chapter 3
Section 3.1

Chapter 4
Section 4.1

etc

Is it possible to achieve? If yes, how?

**Comment by denismaier** [comment]
Maybe have a look into reference files.

Von: Andriy Burkov ***@***.***>
Gesendet: Mittwoch, 31. Juli 2024 05:49
An: jgm/pandoc ***@***.***>
Cc: Subscribed ***@***.***>
Betreff: [jgm/pandoc] Parts and chapters independent on one another (Discussion #10040)


Hi!

I'm using Pandoc with rendering into docx. I would like to organize my chapters into several parts. Parts would have numbers like 1, 2, 3, etc. I would like chapters to have numbers independent from parts. Like this:

Part 1

Chapter 1
Section 1.1

Chapter 2
Section 2.1

Part 2

Chapter 3
Section 3.1

Chapter 4
Section 4.1

etc

Is it possible to achieve? If yes, how?

-

**Comment by jgm** [comment]
sorry, no

**Comment by jgm** [comment]
You could try marking the top-level sections with the `{.unnumbered}` class, and inclding the number directly:

```
# Part 1 {.unnumbered}

## Chapter
```

Not sure if this will work properly, but try it.

**Comment by jpcirrus** [comment]
Possibly, the [parts-and-chapters](https://github.com/tarleb/parts-and-chapters) filter is what you are looking for.

**Reply by aburkov** [reply]
> Possibly, the [parts-and-chapters](https://github.com/tarleb/parts-and-chapters) filter is what you are looking for.

This works alone, but it conflicts with cross-refer filter. If I use parts-and-chapters before cross-refer, cross references are broken. Otherwise parts-and-chapters don't work :-(

**Reply by jpcirrus** [reply]
> This works alone, but it conflicts with cross-refer filter. If I use parts-and-chapters before cross-refer, cross references are broken. Otherwise parts-and-chapters don't work :-(

I'm not aware of a cross-refer filter. Do you mean [pandoc-crossref](https://github.com/lierdakil/pandoc-crossref)? Have you ensured that the settings for pandoc, parts-and-chapters and pandoc-crossref re section numbering are in harmony? I haven't yet used the parts-and-chapters filter as it was only released 2 weeks ago so can't help you there.
